Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "redis-queue"
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
Please bump to 0.3.0 as redis change maybe big change

Reply:
Will do